builtin/game/chatcommands.lua

```diff
@@ -718,7 +718,7 @@ core.register_chatcommand("rollback", {
 	description = "Revert actions of a player. Default for <seconds> is 60",
 	privs = {rollback=true},
 	func = function(name, param)
-		if not core.setting_getbool("enable_rollback_recording") then
+		if not core.settings:get_bool("enable_rollback_recording") then
```

Comment:
No, I removed `settings_getbool` and replaced it with the existing Settings's `get_bool`.  This is just a preexisting inconsistency between the names of two different functions that I switched between.